app/controllers/v0/profile/personal_informations_controller.rb

```diff
@@ -39,21 +39,15 @@ def raise_error!
       end
 
       def log_errors_for(response)
-        if response.gender.nil? || response.birth_date.nil?
-          sanitized_response = sanitize_data(response.to_h)
-          sanitized_params = sanitize_data(params)
-          log_message_to_sentry(
-            'mpi is missing data bug:',
-            :info,
-            {
-              response: sanitized_response,
-              params: sanitized_params,
-              gender: response.gender,
-              birth_date: response.birth_date
-            },
-            profile: 'pciu_profile'
-          )
-        end
+        return unless response.gender.nil? || response.birth_date.nil?
+
+        Rails.logger.error("mpi missing data: #{I18n.t('common.exceptions.MVI_BD502.detail')}", {
+                             response:,
+                             params:,
+                             gender: response.gender,
+                             birth_date: response.birth_date,
+                             mvi_status_code: I18n.t('common.exceptions.MVI_BD502.status')
+                           })
```

Comment:
changing the level from `info` to `error` as well as switching away from Sentry.